stripe/stripe_object.py

```diff
@@ -417,3 +434,12 @@ def __deepcopy__(self, memo):
             super(StripeObject, copied).__setitem__(k, deepcopy(v, memo))
 
         return copied
+
+    _inner_class_types: ClassVar[Dict[str, Type["StripeObject"]]] = {}
```

Comment:
Optional/NonRequired?

Reply:
No, required but always defined on the parent class leaves fewer edge cases, I think.